config.yml

```diff
@@ -145,6 +145,9 @@ collections:
             search_fields: ['slug', 'name']
             value_field: '{{slug}}'
             display_fields: ['{{slug}}: {{name}}']
+          - label: 'Return path'
```

Comment:
This is more of a for me later type of question, but what do we expect in the return path?

Reply:
We are expecting a list of object keys separated by dots. 
e.g. `hasIAMRoles.hasIAMPolicy.hasIAMPolicyStatement.hasIAMPermission.isOwnedByIAMAssetType.includesKMSKey`

Reply:
Maybe we could use a [`hint`](https://www.netlifycms.org/docs/widgets/#:~:text=defaults%20to%20true-,hint,-%3A%20optionally%20add%20helper)